config/crd/bases/fluentbit.fluent.io_clusterinputs.yaml

```diff
@@ -156,6 +156,42 @@ spec:
                       plugin.
                     type: string
                 type: object
+              opentelemetry:
```

Comment:
Still opentelemetry , You'll need to regenerate the yaml by `make test`
And the `opentelemetry` in the helm chart needs to be changed to `openTelemetry `

Reply:
Thanks,  I have done it.